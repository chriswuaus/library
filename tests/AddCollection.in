add collection aa.csv
ADD collection yeehaw.csv
add collection test2.csv
add collection test3nodup.csv
add collection test.csv
add collection test.csv
exit
