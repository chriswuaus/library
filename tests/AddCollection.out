user: 1 books successfully added.

user: No such collection.

user: 12 books successfully added.

user: 4 books successfully added.

user: No books have been added to the system.

user: No books have been added to the system.

user: Ending Library process.
